pycode/memilio-simulation/memilio/simulation/abm.cpp

```diff
@@ -82,7 +82,7 @@ PYBIND11_MODULE(_simulation_abm, m)
     pymio::bind_ParameterSet<mio::abm::LocalInfectionParameters>(m, "LocalInfectionParameters").def(py::init<>());
     pymio::bind_ParameterSet<mio::abm::MigrationParameters>(m, "MigrationParameters").def(py::init<>());
 
-    py::class_<mio::abm::TimeSpan>(m, "TimeSpan")
+    pymio::bind_class<pymio::PickleFlag::TryPickling, mio::abm::TimeSpan>(m, "TimeSpan")
```

Comment:
why wouldn't this be ForcePickling?

Reply:
It probably should be in the future, but for now that results in a compilation error, because TimeSpan is not serializable.